index.js

```diff
@@ -1,6 +1,7 @@
 var search = require('./search')
+var generateGrams = require('./generateGrams')
 
-module.exports = function fuzzyfind (input, collection, accessor) {
+module.exports = function fuzzyfind (input, collection, accessor, precision) {
```

Comment:
What are your thoughts on allow accessor and precision to be optional  alone?

for instance:

~~~ javascript
fuzzyfind(input, collection) // already works
fuzzyfind(input, collection, accessor) // already works
fuzzyfind(input, collection, accessor, precision) // already works
fuzzyfind(input, collection, precision) // weee!!!
~~~

This is really common in [node](https://nodejs.org/api/fs.html#fs_fs_readfile_file_options_callback)